ratis-logservice/src/test/java/org/apache/ratis/logservice/server/TestMetaServer.java

```diff
@@ -123,6 +129,41 @@ public void testReadWritetoLog() throws IOException, InterruptedException {
         assert(res.array().length > 0);
     }
 
+    @Test
+    public void testLogArchival() throws IOException, InterruptedException {
```

Comment:
Can you add some more tests, please?

Should have an e2e UT how this would work for users. Could set a local filesystem path (e.g. file:///..) and then use the LogServiceClient to trigger the archive of the log. Then, validate that the file can read from the archive'd log path.

Same goes for the export use-case.

Reply:
sure let me add some more tests, but this is actually e2e test, probably you have seen this config later.
https://github.com/apache/incubator-ratis/pull/25#discussion_r299662354